src/components/NavBar/index.js

```diff
@@ -89,50 +110,54 @@ class NavBar extends React.PureComponent {
       match,
     } = this.props;
 
-    if (this.props.match.path === currentPath) {
-      return (
-        <img
-          src="/images/nav-arrow.png"
-          alt="nav-arrow"
-          className={
-            classNames(
-              classes.navArrow,
-              currentPath === '/my-wallet' || currentPath === '/activities' ? 'right' : ''
-            )
-          }
-        />
-      );
-    }
-
-    return null;
+    return (
+      <img
+        src="/images/nav-arrow.png"
+        alt="nav-arrow"
+        className={
+          classNames(
+            classes.navArrow,
+            currentPath === RouterPath.myWallet || currentPath === RouterPath.set ? 'right' : ''
```

Comment:
there's a better way to do this. @dwalintukan 
```
classNames(classes.navArrow, {
  right: currentPath === RouterPath.myWallet || currentPath === RouterPath.set
})
```

Reply:
so the { } will return undefined? or will it be `right: undefined`?